testscommon/webSocketReceiverStub.go

```diff
@@ -2,12 +2,18 @@ package testscommon
 
 import (
 	"github.com/multiversx/mx-chain-core-go/webSocket"
+	"github.com/multiversx/mx-chain-core-go/webSocket/data"
 )
 
 // WebSocketReceiverStub -
 type WebSocketReceiverStub struct {
 }
 
+// Send -
+func (w WebSocketReceiverStub) Send(_ data.WsSendArgs, _ webSocket.WSConClient) error {
```

Comment:
Please make this a proper stub & add pointer receivers

Reply:
done